hivemind/dht/schema.py

```diff
@@ -100,6 +108,14 @@ def _key_id_to_str(key_id: bytes) -> str:
 
         return binascii.hexlify(key_id).decode()
 
+    def merge_with(self, other: RecordValidatorBase) -> bool:
+        if not isinstance(other, SchemaValidator):
+            return False
+
+        self._alias_to_name.update(other._alias_to_name)
```

Comment:
It is no longer a private field

Reply:
In strictly typed languages, accessing a private field of another instance of the same class is valid (e.g. in C++, see "Private member access" [here](https://en.cppreference.com/w/cpp/language/access)), so I think it's reasonable to assume the same in Python.

Reply:
Agreed!